set-quotas.py

```diff
@@ -51,8 +51,22 @@ def parse_rows(rows):
     # NOTE: entry[17] is a required field in the Google Form, so it is safe
     # to assume entry[0:16] exists.
 
+    PROJECT_COLUMN = 7
     project_list = []
-    for idx, entry in enumerate(rows):
+
+    if select_project:
+        try:
+            rows = select_rows(select_project, PROJECT_COLUMN, rows)
+            if len(rows) > 2:
+                print ("WARNING: Multiple requests found for project {}. All "
+                       "{} requests will be processed. You may need to close "
+                       "multiple tickets.").format(args.project, len(rows))
```

Comment:
there's a header row, so the # of requests that will be processed is 1 less than the length of the rows.

This is for the edge case where someone filled out the form multiple times in a row, we're alerting the helpdesk how many tickets to look for.  There is probably a better way to handle this, but for now at least they will know.